### ISSUE TITLE
sea ice

### ISSUE THREAD

**Issue by PaulSpence** [adele157/IAF-meltwater#7]
what happens

**Comment by PaulSpence** [comment]
<img width="564" alt="Screen Shot 2022-08-17 at 6 25 04 pm" src="https://user-images.githubusercontent.com/6644956/185271761-98dc5b3c-516d-431c-9d14-6857e7d3dbb7.png">

**Comment by PaulSpence** [comment]
<img width="476" alt="Screen Shot 2022-08-17 at 6 29 14 pm" src="https://user-images.githubusercontent.com/6644956/185272238-cb467c9b-697d-47fd-b231-28af69fdf240.png">

**Comment by adele157** [comment]
This is a smaller effect than I was expecting. 

Is the increasing sea ice in the Ross due to increased stratification and surface cooling there (i.e. associated with a decrease in dense water formation)?

And is the West Antarctic decrease in sea ice related to the ocean warming there (though we have only looked at bottom temperature there not SST trends so far)?

Might be worth plotting sea ice concentration anomaly also?

**Comment by matthew-england-unsw** [comment]
Yes agreed - plots of SSS and SST would assist here.   I added the SSS as an issue elsewhere (to check MW pathways mainly, as I reckon that's a really interesting thing to check regardless, to see where the MW accumulates).   Maybe we should merge under a header issue "Surface T-S and sea-ice response".

**Comment by adele157** [comment]
This is consistent with the [SST change](https://github.com/adele157/IAF-meltwater/issues/10#issuecomment-1233587108). Let's keep issues separate, otherwise it gets messy. We can link between them.

**Comment by PaulSpence** [comment]
![Screen Shot 2022-09-15 at 11 08 20 am](https://user-images.githubusercontent.com/6644956/190289731-91c20ce3-c880-4ffb-a53b-7e617e838025.png)

**Comment by PaulSpence** [comment]
![Screen Shot 2022-09-26 at 2 06 15 pm](https://user-images.githubusercontent.com/6644956/192192365-8ffad65b-de6f-4890-bde7-e5d6eac4fb5c.png)

Sea ice growth due to thermal and dynamic properties. Note thermal tends to oppose dynamics since sea ice needs to diverge in order for new growth to occur. I thought we would see more convergence growth where there is the greatest thickness change (west of ross sea).

**Comment by PaulSpence** [comment]
Decadal thickness anomalies:

![Screen Shot 2022-09-26 at 3 09 39 pm](https://user-images.githubusercontent.com/6644956/192198148-998daff7-5212-403d-b214-db9d6df9bf7d.png)

**Comment by PaulSpence** [comment]
Surface ocean velocities:
 
![Screen Shot 2022-09-26 at 3 11 13 pm](https://user-images.githubusercontent.com/6644956/192198283-49e9e57f-5844-4705-b262-e8da309b53e0.png)

**Comment by PaulSpence** [comment]
Frazil growth:
![Screen Shot 2022-09-26 at 3 14 16 pm](https://user-images.githubusercontent.com/6644956/192198598-68987478-faeb-4d76-ad3a-46f8bf7ad2f7.png)

**Comment by PaulSpence** [comment]
Overall:
i) Meltwater increase the coastal currents in Amundsen region
ii) Current anomaly moves ice westward, allowing for more frazil growth in Amundsen region (and congel growth further downstream)
iii) Tend to lose sea ice thickness in the Amundsen and gain it in west Ross as it gets piled up downstream

**Comment by PaulSpence** [comment]
Monthly sea Ice Volume time-series: Melt anomaly relative to Control and Control relative to monthly climatology.
![Screen Shot 2022-09-26 at 3 14 16 pm](https://user-images.githubusercontent.com/6644956/192199014-f9a812bf-48ba-425c-99cd-60b9c7a1055f.png)

Modest but not insignificant gains in ice volume in Ross and loss in Amundsen.

**Comment by PaulSpence** [comment]
Sea Ice Area anoms compared to obs:
![Screen Shot 2022-09-26 at 3 20 01 pm](https://user-images.githubusercontent.com/6644956/192199288-d1bf165a-e352-41f8-8556-fc673dd5d4e7.png)
